src/globalTypes.ts

```diff
@@ -0,0 +1,17 @@
+import { i18n } from '@lingui/core';
```

Comment:
Create a folder named global-types have a file named evaluation.ts put these enum and evaluation dictionary in it, create a types.d.ts index the folder and re export these